configs/sst_virtualization-virt.yaml

```diff
@@ -0,0 +1,97 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: Virtualization on RHEL - KVM Userspace
+  description: Userspace agents and tools that enable virtualization capabilities on RHEL
+  maintainer: sst_virtualization
+
+  modules_enable:
+  - virt:rhel
+
+  packages:
+  - amtterm
+  - augeas
+  - dtc
+  - edk2
+  - fakechroot
+  - fakeroot
+  - fuse-sshfs
```

Comment:
I've long wanted `fuse-sshfs` to be in RHEL, championing it since RHEL 5, but we never managed to get it in.  It's something that lots of customers want though.  However it's not a virt module package, probably AppStream.